workers/unity/Packages/com.improbable.gdk.health/Systems/HealthRegenSystem.cs

```diff
@@ -1,54 +1,71 @@
+using System.Collections.Generic;
 using Improbable.Gdk.Core;
-using Improbable.Gdk.ReactiveComponents;
-using Unity.Collections;
+using Improbable.Worker.CInterop;
```

Comment:
Why is this needed?

Reply:
For the `Authority.NotAuthoritative` on line 93